interface-definitions/include/auth_mode_pre_shaerd_secret.xml.i

```diff
@@ -0,0 +1,14 @@
+<!-- include start from auth_mode_pre_shaerd_secret.xml.i -->
```

Comment:
Typo `shaerd` in the xml name and include name

Reply:
fixed